crates/orderbook/openapi.yml

```diff
@@ -1538,8 +1538,20 @@ components:
           type: number
       required:
         - factor
+    PriceImprovement:
+      description: The protocol fee is taken as a percent of the order price improvement which is a difference between the executed price and the best quote.
```

Comment:
nit: Should we specify the range of the values [0, 1)?

Reply:
Added